### PR TITLE
[GroupMgr] Group model change

### PR DESCRIPTION
**Pull request by Jwaegebaert**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: certified-connectors/GroupMgr/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,982 +1,987 @@
...
+    },
+    "host": "vragroupmanagerapprovalflow-h8tvt-uzi02fbsspubp3eg.azurewebsites.net",
+    "basePath": "/",
+    "schemes": [
+        "https"
+    ],
+    "x-ms-connector-metadata": [{
+            "propertyName": "Website",
+            "propertyValue": "https://groupmgr.com/"
+        }, {
```
Comment (sriyen-msft) [inline certified-connectors/GroupMgr/apiDefinition.swagger.json:21]: @Jwaegebaert Please fix indentation issues in the swagger as properties are not aligned properly.

File: certified-connectors/GroupMgr/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1,982 +1,987 @@
...
+        "title": "GroupMgr",
+        "description": "A powerful set of tools, for both IT managers and users, to efficiently create, manage, browse and analyse all groups in your Microsoft 365 environment.",
+        "version": "2.5",
+        "contact": {
+            "name": "GroupMgr Helpdesk",
+            "url": "https://www.vanroey.be/en/support/",
+            "email": "support@groupmgr.com"
+        }
+    },
+    "host": "vragroupmanagerapprovalflow-h8tvt-uzi02fbsspubp3eg.azurewebsites.net",
```
Comment (sriyen-msft) [inline certified-connectors/GroupMgr/apiDefinition.swagger.json:13]: @Jwaegebaert Can you please make sure that you are providing a valid host URL as we do not certify a connector with test or dev host URL.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/GroupMgr/apiDefinition.swagger.json (hunk 2 above)
Comment (Jwaegebaert) [reply certified-connectors/GroupMgr/apiDefinition.swagger.json:13]: This URL is the one from production. It isn't a test or dev URL.

File: certified-connectors/GroupMgr/apiDefinition.swagger.json (hunk 1 above)
Comment (Jwaegebaert) [reply certified-connectors/GroupMgr/apiDefinition.swagger.json:21]: @sriyen-msft  Could you elaborate a bit more about this issue. The selected part above looks like the indentations are properly aligned.

**Comment by sriyen-msft** [comment]
@Jwaegebaert I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by Jwaegebaert** [comment]
@sriyen-msft Hi, I recently commentend on your suggestion about indentation issues. I didn't really notice any issue on the suggested code part. Is it possible to elaborate a bit more about this issue?

File: certified-connectors/GroupMgr/apiDefinition.swagger.json (hunk 1 above)
Comment (sriyen-msft) [reply certified-connectors/GroupMgr/apiDefinition.swagger.json:21]: @Jwaegebaert Each and of property of each section in the swagger should be properly aligned with tab index. If you see this section, the brackets are not properly aligned as the opening and closing braces aligned on the same line. You can refer [a connector](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/SAS%20Decisioning/apiDefinition.swagger.json#L13) in the open source repo for reference.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/GroupMgr/apiDefinition.swagger.json (hunk 1 above)
Comment (Jwaegebaert) [reply certified-connectors/GroupMgr/apiDefinition.swagger.json]: @sriyen-msft Thanks for the elaboration. The changes are commited.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**